sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/ListContainersIncludeTypeWrapper.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.blob.implementation;
```

Comment:
Should this and `ListBlobContainersIncludeTypeWrapper` both exist? Should this be removed and the XML node names be fixed in the `BlobContainers` variant?

Reply:
Good point. Ideally both are not needed.
However, I didn't find a way to force generator to skip `ListBlobContainersIncludeTypeWrapper`, so I end result has just one wrapper.

I can see that similar issue exist for existing `ListBlobsIncludeItemWrapper` and `ListBlobsIncludeItem` pair - the wrapper is not used but exist.